### PR TITLE
Feature/compare countries chosen

### PR DESCRIPTION
**Pull request by paulahaertel**
- adds toolbar as layout/moudule
- adds chosen.js dropdown to country compare

### CODE REVIEW

File: app/assets/javascripts/compare/views/index/CompareSelectorsView.js [hunk 1]
```diff
@@ -61,10 +71,42 @@ define([
...
+      this.status.set(selector, country);
+    },
+
+    _disableSelectors: function(country, selector) {
+      var selectors = [country1, country2, country3];
+      var index = selectors.indexOf(selector);
+
+      selectors.splice(index, 1);
+
+      console.log()
```
Comment (apercas) [inline app/assets/javascripts/compare/views/index/CompareSelectorsView.js]: Hey, would you mind cleaning this? :)

**Comment by apercas** [comment]
@paulahaertel Clara's on duty! :dancers:

File: app/assets/stylesheets/modules/common/_m-toolbar.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.m-toolbar {
+  display: inline-block;
+  width: 100%;
+  .options {
+    display: inline-flex;
+    align-items: center;
+    float: right;
+    .options_item {
+      float: left;
```
Comment (dhakelila) [inline app/assets/stylesheets/modules/common/_m-toolbar.scss]: When inside of a flex item, float doesn't work.

File: app/views/compare/index.html.slim [hunk 3]
```diff
@@ -10,17 +10,31 @@ section.l-compare-countries
...
     .inner
+      .m-toolbar
+        ul.options
+          li.options_item
+            svg
+              use (xlink:href="#shape-share")
+            button#share-country share
+          li.options_item
+            svg
+              use (xlink:href="#shape-share")
```
Comment (dhakelila) [inline app/views/compare/index.html.slim]: Print icon is different.

File: app/assets/javascripts/compare/views/index/CompareSelectorsView.js [hunk 4]
```diff
@@ -54,6 +55,15 @@ define([
...
       this._stopSpinner();
+
+      $('#country1').chosen();
+      $('#country2').chosen();
+      $('#country3').chosen();
+
+      $('#country1').on('change', function(){
+        console.log($("#country1").chosen().val());
+      });
+
```
Comment (dhakelila) [inline app/assets/javascripts/compare/views/index/CompareSelectorsView.js]: I think, you don't really need this.

File: app/assets/javascripts/compare/views/index/CompareSelectorsView.js [hunk 5]
```diff
@@ -54,6 +55,15 @@ define([
       this.$el.html(this.template({'countries': countries}))
 
       this._stopSpinner();
+
+      $('#country1').chosen();
+      $('#country2').chosen();
+      $('#country3').chosen();
+
```
Comment (dhakelila) [inline app/assets/javascripts/compare/views/index/CompareSelectorsView.js]: I think, it is better if you separate this into another function.

**Comment by dhakelila** [comment]
Good job!

**Comment by apercas** [comment]
Wohoooo!!!1!11ONE

**Pull request merged**